src/SearchQuery.php

```diff
@@ -2,69 +2,47 @@
 
 namespace CultuurNet\SearchV3;
 
-use Guzzle\Http\Client;
-use CultureFeed_HttpClient;
-use CultureFeed_HttpResponse;
-use CultureFeed_DefaultHttpClient;
-use Exception;
-use Guzzle\Http\ClientInterface;
-use Guzzle\Http\Exception\BadResponseException;
-use Guzzle\Http\Message\EntityEnclosingRequestInterface;
-use Guzzle\Service\Builder\ServiceBuilderInterface;
-
-class SearchQuery implements SearchQueryInterface
+final class SearchQuery implements SearchQueryInterface
 {
-
     /**
      * @var ParameterInterface[]
      */
-    protected $parameters = [];
+    private $parameters = [];
 
     /**
      * @var array
      */
-    protected $sorting = [];
+    private $sorting = [];
 
     /**
      * Return the full embedded search results, or only the ids.
      * @var bool
      */
-    protected $embed = false;
+    private $embed = false;
```

Comment:
PHPStorm warns that this property is overridden in the constructor.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/9268fd6f232389803cf606f39342d6fe054878a3